src/css/components/menu.less

```diff
@@ -15,7 +15,7 @@
       border-right: transparent 2px solid;
       opacity: .8;
       &:hover {
-        border-color: currentColor;
+        border-color: @modal-footer-border-color;
```

Comment:
Please introduce a dedicated menu variable instead of reusing the `@modal-footer-border-color`.